integ-test/src/test/java/org/opensearch/sql/calcite/standalone/CalcitePPLJoinIT.java

```diff
@@ -68,7 +69,7 @@ public void testJoinWithCondition() {
         schema("state", "string"),
         schema("country", "string"),
         schema("occupation", "string"),
-        schema("country0", "string"),
+        schema("b.country", "string"),
```

Comment:
IMO, we should have unified rules to handle duplicated columns. 2 options in my mind
1. similar as lookup command, using APPEND and REPLACE options to handle duplicate columns. 
2. Or follow SQL standard way, then column name of  a.country should be a.country instead of country.

Reply:
what if user does not specify fields command, what is expected output?

Reply:
> we should have unified rules to handle duplicated columns. IMO, 2 options
> 
> 1. similar as lookup command, using APPEND and REPLACE options to handle duplicate columns.
> 2. Or follow SQL standard,  user must explicitly qualify or alias the columns, then, the column name of  a.country should be a.country instead of country.

1. The outputs of `lookup` doesn't contain duplicated fields. I think `lookup` and `join` are different.
2. `user must explicitly qualify or alias the columns` do you mean access them? Yes for now, user must explicitly specific the `id` field by `alias.id` or `tablename.id`, else an ambiguous exception shows. Without this patch, `alias.id` or `tablename.id` cannot work since the `id` renamed to `id0`. It just impacts the output schema.